### PR TITLE
add text size widget

### PR DESCRIPTION
**Pull request by mjhea0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +38,30 @@ Import utils like so:
...
+import Vue from "vue";
+import Vuex from "vuex";
+import App from "./App.vue";
+import { scaifeWidgets } from "@scaife-viewer/scaife-widgets";
+
+Vue.use(Vuex);
+
+const store = new Vuex.Store({
+  modules: {
+    scaifeWidgets
```
Comment (jhrr) [inline README.md]: Is this shorthand equivalent to [this](https://github.com/scaife-viewer/explorehomer/blob/scaife/widgets/src/store.js#L20)?

File: src/store/constants.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/prefer-default-export
+export const MODULE_NS = "scaifeWidgets";
```
Comment (jhrr) [inline src/store/constants.js]: I'm sure you already probably have it on your `TODO` list but we need to make sure to export this from the package too so other modules can access the 'namespaced-state-space'.

File: README.md (hunk 1 above)
Comment (mjhea0) [reply README.md]: yes

Comment (jhrr) [reply README.md]: cool! didn't see it done that way anywhere else, handy.

Comment (mjhea0) [reply README.md]: Well, it only works when the module name is the same as the variable - `scaifeWidgets`.

Comment (jhrr) [reply README.md]: ah, if the `namespace` name is the same? i get you, it's just regular object shorthand. still a nice shortcut.

Comment (mjhea0) [reply README.md]: Version 1:

```js
const scaifeWidgets = {
  namespaced: true,
  state,
  actions,
  getters,
  mutations
};

const store = new Vuex.Store({
  modules: { scaifeWidgets }
});
```

Version 2:

```js
const scaifeWidgets = {
  namespace: MODULE_NS,
  store: {
    namespaced: true,
    state,
    actions,
    getters,
    mutations
  }
};

...

const store = new Vuex.Store({
  modules: {
    [scaifeWidgets.namespace]: scaifeWidgets.store
  }
});
```

Version 3:

```js
const scaifeWidgets = {
  namespace: MODULE_NS,
  store: {
    namespaced: true,
    state,
    actions,
    getters,
    mutations
  }
};

const module = {};
module[scaifeWidgets.namespace] = scaifeWidgets.store
const store = new Vuex.Store({
  modules: module
});
```

Comment (mjhea0) [reply README.md]: Depends on how readable you like your code...

Comment (mjhea0) [reply README.md]: #2 is what I see most. I find it hard to read `[scaifeWidgets.namespace]`, though...

Comment (jhrr) [reply README.md]: right, so you are eliding the `store` to just splat it all in via the matching names.

I guess #2 is the most common, as you say, as it's the way it's done in the Vue docs. Ultimately I think we just need to be consistent and I think `scaife-skeleton` follows #2, so we'd have to change the shape of that module's data too if we go with anything different.

**Pull request merged**